odoo/tools/translate.py

```diff
@@ -1621,6 +1633,8 @@ def _get_translation_upgrade_queries(cr, field):
             }
             if "en_US" not in new_values:
                 new_values["en_US"] = field.translate(lambda v: None, src_value)
+            if website_installed and len(record) == 5 and record[4] not in new_values:
```

Comment:
no need to check `website_installed` here, if not installed, the `len(record) == 4`

Reply:
Changes done